DESCRIPTION

```diff
@@ -25,10 +25,11 @@ Suggests:
     dplyr,
     knitr,
     lifecycle,
-    testthat
+    testthat (>= 3.0)
 VignetteBuilder: 
     knitr
 Config/Needs/website: insightsengineering/nesttemplate
+Config/testthat/edition:3
```

Comment:
This is not something that should be happening as an aside as part of another PR its a (much) larger discussion that should be had separately, and certainly not this close before release